src_py/apiServer/apiServer.py

```diff
@@ -43,13 +58,21 @@ def __init__(self):
         self.transmitter = Transmitter(self.mainServerAddress)
 
         print("\n***Please remember to execute NerlnetRun.sh before continuing.")
-        
+
     def sendJsonsToDevices(self):
         # Send the content of jsonPath to each devices:
         print("\nSending JSON paths to devices...")
 
-        archAddress = globe.content[0][:-1]
-        connMapAddress = globe.content[1][:-1]
+        # archAddress = globe.content[0][:-1]
+        # connMapAddress = globe.content[1][:-1]
+
+        archAddress , connMapAddress, exp_flow_json = self.getUserJsons()
+        [bad, archPath] = archAddress.split("/NErlNet")
+        archAddress = "../../.."+archPath
```

Comment:
What is the hidden assumption of going 3 directories up?
I don't like this approach.
Please put an explanation here or get rid of these 3 ".."

Reply:
added explanation